### ISSUE TITLE
how to replace NMS with a different version

### ISSUE THREAD

**Issue by traderandreas** [ZFTurbo/Keras-RetinaNet-for-Open-Images-Challenge-2018#5]
Hi ZFTurbo :) I have written an algorithm to replace NMS, and implemented it on FastAi. I've been trying to implement it in the original keras-retinanet, but I'm running into a lot of issues. I cannot even set NMS to False, let alone make changes to the code. I have installed the repo and scripts, and compiled changes in PyCharm and on the command line, but the value of NMS remains set to True. Do you have any ideas as to what I may be doing wrong? How would I make changes to the code in your repo?

**Comment by ZFTurbo** [comment]
Can you please add more details of what are you trying to do.

**Comment by traderandreas** [comment]
Hi zfturbo I have my own version of nms  and I want to replace it with my
code. What ever change I make in the original retinanet code the changes
are not seen?

Also I would like to detect farm animals like pigs cows etc and humans
could you please advise me how I can use transfer learning to do this
without training on humans etc etc.

Your help would be greatly appreciated.

Andreas

**Comment by ZFTurbo** [comment]
May be this question is more suitable for keras-retinanet repository:
https://github.com/fizyr/keras-retinanet

To train on different labels you can use pretrained model from this page. Check FAQ how to do so:
https://github.com/fizyr/keras-retinanet#faq
